certbot/storage.py

```diff
@@ -759,7 +783,7 @@ def new_lineage(cls, lineagename, cert, privkey, chain, cli_config):
         """
 
         # Examine the configuration and find the new lineage's name
-        for i in (cli_config.renewal_configs_dir, cli_config.archive_dir,
+        for i in (cli_config.renewal_configs_dir, cli_config.default_archive_dir,
```

Comment:
Does this make it impossible to specify an archive directory when creating a new lineage? So would the only path to creating a new lineage with a specified archive directory be to first created it and then update the symlinks with `_update_symlinks()` after the lineage is finished being created?